server/src/utils/getConfig.ts

```diff
@@ -12,36 +12,84 @@ import {
   VV_SCREENSHARE_ENABLED_ENV_NAME,
   VV_WAITING_SUBTITLE_ENV_NAME,
   VV_WAITING_TITLE_ENV_NAME,
-  VV_LOGO_URL_ENV_NAME
+  VV_LOGO_URL_ENV_NAME,
+  VV_POSTCALL_SURVEY_TYPE_ENV_NAME,
+  VV_POSTCALL_SURVEY_OPTIONS_SURVEYURL_ENV_NAME
 } from '../constants';
 
-import { ServerConfigModel, ClientConfigModel } from '../models/configModel';
-import DefaultConfig from '../defaultConfig.json';
+import { ServerConfigModel, ClientConfigModel, PostCallSurveyType, PostCallConfig } from '../models/configModel';
+import { getDefaultConfig } from './getDefaultConfig';
 
 export const getServerConfig = (): ServerConfigModel => {
-  return {
+  const defaultConfig = getDefaultConfig();
+
+  const config = {
     communicationServicesConnectionString:
-      process.env[VV_COMMUNICATION_SERVICES_CONNECTION_STRING] ?? DefaultConfig.communicationServicesConnectionString,
-    microsoftBookingsUrl: process.env[VV_MICROSOFT_BOOKINGS_URL_ENV_NAME] ?? DefaultConfig.microsoftBookingsUrl,
+      process.env[VV_COMMUNICATION_SERVICES_CONNECTION_STRING] ?? defaultConfig.communicationServicesConnectionString,
+    microsoftBookingsUrl: process.env[VV_MICROSOFT_BOOKINGS_URL_ENV_NAME] ?? defaultConfig.microsoftBookingsUrl,
     chatEnabled:
       typeof process.env[VV_CHAT_ENABLED_ENV_NAME] === 'string'
         ? process.env[VV_CHAT_ENABLED_ENV_NAME]?.toLowerCase() === 'true'
-        : DefaultConfig.chatEnabled,
+        : defaultConfig.chatEnabled,
     screenShareEnabled:
       typeof process.env[VV_SCREENSHARE_ENABLED_ENV_NAME] === 'string'
         ? process.env[VV_SCREENSHARE_ENABLED_ENV_NAME]?.toLowerCase() === 'true'
-        : DefaultConfig.screenShareEnabled,
-    companyName: process.env[VV_COMPANY_NAME_ENV_NAME] ?? DefaultConfig.companyName,
-    colorPalette: process.env[VV_COLOR_PALETTE_ENV_NAME] ?? DefaultConfig.colorPalette,
-    waitingTitle: process.env[VV_WAITING_TITLE_ENV_NAME] ?? DefaultConfig.waitingTitle,
-    waitingSubtitle: process.env[VV_WAITING_SUBTITLE_ENV_NAME] ?? DefaultConfig.waitingSubtitle,
-    logoUrl: process.env[VV_LOGO_URL_ENV_NAME] ?? DefaultConfig.logoUrl
-  };
+        : defaultConfig.screenShareEnabled,
+    companyName: process.env[VV_COMPANY_NAME_ENV_NAME] ?? defaultConfig.companyName,
+    colorPalette: process.env[VV_COLOR_PALETTE_ENV_NAME] ?? defaultConfig.colorPalette,
+    waitingTitle: process.env[VV_WAITING_TITLE_ENV_NAME] ?? defaultConfig.waitingTitle,
+    waitingSubtitle: process.env[VV_WAITING_SUBTITLE_ENV_NAME] ?? defaultConfig.waitingSubtitle,
+    logoUrl: process.env[VV_LOGO_URL_ENV_NAME] ?? defaultConfig.logoUrl
+  } as ServerConfigModel;
+
+  config.postCall = getPostCallConfig(defaultConfig);
+
+  return config;
+};
+
+const isValidPostCallSurveyType = (postcallSurveyType: string): postcallSurveyType is PostCallSurveyType => {
+  return ['msforms', 'thirdparty'].indexOf(postcallSurveyType) !== -1;
+};
+
+const getPostCallConfig = (defaultConfig: any): PostCallConfig | undefined => {
```

Comment:
I am not sure if lint complain about this, but I am not sure it is good idea to leave `any` type for `defaultConfig` parameter.

Since we have `ServerConfigModel` interface and `postCall` property is optional, we can replace `any` to `ServerConfigModel`?

Reply:
made this change and pushed . Thanks!